blog/2023-06-14-what-llm-post.md

```diff
@@ -0,0 +1,51 @@
+---
+slug: what-is-llm
+title: What is an LLM? 
+authors:
+  name: Darpan Ganatra 
+  title: What is an LLM? 
+  url: https://github.com/DarpanGanatra
+tags: [LLM]
+---
+**Large Language Models (LLMs)** are machine learning models trained to understand and generate human-like text. They are a type of transformer-based model that is designed to predict the next word in a sentence, given the preceding context, also known as autoregressive models. Notable examples include GPT-3, GPT-4, and similar transformer models.
```

Comment:
I don't think you need to include "type of transformer base model" they don't know what that is. And for the prediction, I think it could be good to include the fact they have no sense of "the truth", purely just responding on what they think is most likely to be the answer.